int32 label_id
float64[] position
float64[] size
bool[] view_angles